app/controllers/widgets_controller.rb

```diff
@@ -27,8 +27,11 @@ def set_web_widget
 
   def check_domain
     return if Rails.env.development?
-    return if request.headers['Referer'].downcase.start_with? @web_widget.website_url.downcase
-
+    # support multiple domains from same referer header param
+    domains = @web_widget.website_url.downcase.split("|")
```

Comment:
Is this ok? IDK if there are standard practices in web security regarding the referers, but usually there's only one.

Reply:
I didn't see any best practice respect of that. 🤔 
I will merge this as is, in the meantime, if we find any argument agains this I will remove it back.